x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/profile/SecurityDomainIntegTests.java

```diff
@@ -153,23 +322,4 @@ public void testDomainCaptureForServiceToken() {
             .execute()
             .actionGet();
     }
-
-    private void assertDomainCapturedForToken() {
-        final SearchResponse searchResponse = client().prepareSearch(SECURITY_TOKENS_ALIAS)
-            .setQuery(
-                QueryBuilders.boolQuery()
-                    .must(QueryBuilders.termQuery("refresh_token.client.user", RAC_USER_NAME))
-                    .must(QueryBuilders.termQuery("refresh_token.refreshed", false))
-            )
-            .execute()
-            .actionGet();
-        assertThat(searchResponse.getHits().getHits(), arrayWithSize(1));
-
-        final XContentTestUtils.JsonMapView responseView = XContentTestUtils.createJsonMapView(
-            new ByteArrayInputStream(searchResponse.getHits().getHits()[0].getSourceAsString().getBytes(StandardCharsets.UTF_8))
-        );
-
-        assertThat(responseView.get("access_token.realm_domain"), equalTo(REALM_DOMAIN_MAP));
-        assertThat(responseView.get("refresh_token.client.realm_domain"), equalTo(REALM_DOMAIN_MAP));
```

Comment:
I don't think we can entirely remove this method and associated test method:
1. The check for domain capture of `access_token` is still needed since it is not related to the refresh token change.
2. Strictly speaking, the domain capture of `refresh_token` also has its place because the new logic for capturing the entire `Authentication` only happens when the min-node version is `onOrAfter(8.2.0)`. This means the domain can still be captured if the cluster is a mix of `8.2.0` and `8.1.0`. This is not really testable with a InternalCluster test. So I am happy for it to be dropped here.
3. One intention of the test is to ensure the tokens (access and refresh) work as normal when the document captures additional domain information. This is still needed since there is no other place that we excercise such a token. Alternatively, you can add it as part of the new test method where you test the refresh behaviour.

Reply:
I have pushed [29925ef](https://github.com/elastic/elasticsearch/pull/85010/commits/29925ef764d82e691e15a56b2b2f3a7c033d1ef3) to test the created access tokens as well as assert the domain conf that gets stored in the document.